Detectors/TOF/workflowIO/src/DigitReaderSpec.cxx

```diff
@@ -35,6 +35,9 @@ void DigitReader::init(InitContext& ic)
   LOG(debug) << "Init Digit reader!";
   auto filename = o2::utils::Str::concat_string(o2::utils::Str::rectifyDirectory(ic.options().get<std::string>("input-dir")),
                                                 ic.options().get<std::string>("tof-digit-infile"));
+
+  mDelayInMuSec = atoi(ic.options().get<std::string>("reader-delay").c_str()) * 1E6;
```

Comment:
`reader-delay` option is dynamically added by the HBFUtilsInitalizer : https://github.com/AliceO2Group/AliceO2/blob/db2b72e6a7868cedbb7ec0d3f6ad7d1568d3d0dc/Detectors/Raw/src/HBFUtilsInitializer.cxx#L79 and triggers a delay in *seconds* between successive injections of TFs (but not before 1st one):
https://github.com/AliceO2Group/AliceO2/blob/db2b72e6a7868cedbb7ec0d3f6ad7d1568d3d0dc/Detectors/Raw/src/HBFUtilsInitializer.cxx#L152-L153
Could you clarify why do you need to add a delay *before* the 1st (and the only one in case of digitization) TF?,

Reply:
because I found that the reader often send the 1st (unique in MC) TF before the QC task is initialized (the reader is too fast).
This means that the task is loosing the single MC TF in many of the subjobs.
Since a MC process has only 1 TF we loose the majority of TFs in TOF digit QC.

By the way, is delay in your initializer in second?
usleep(delay); -> mus?

Cheers,
Francesco

Reply:
also discussing here
https://alice.its.cern.ch/jira/browse/O2-3442